crates/primitives/src/bytecode.rs

```diff
@@ -14,7 +13,7 @@ pub struct JumpMap(pub Arc<BitVec<u8>>);
 impl Debug for JumpMap {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         f.debug_struct("JumpMap")
-            .field("map", &BinaryString::from(self.0.as_raw_slice()))
+            .field("map", &self.0.as_raw_slice())
```

Comment:
Would probably use `hex::encode` here. Forgot the reason why I didn't use that in first place

Reply:
done